README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
1. I dont like "utility classes", because how do you distribute them?
If you include one or two utility classes, do you import the whole utility stylesheet?
(what if other people do the same and they include that stuff - not everyone uses the same things)
...anyway, single modules should be small.

2. I absolutely do not like .is-stateOfComponent, because if somebody defines that class globally and uses your component, it might mess your component and its relatively easy, that somebody will use class names like "is-active"

__Another Question:__
How do you differentiate between a "descendentName" which is managed by the component and a "descendentName" which is managed by a "sub component" with its own namespace?